gradle/libs.versions.toml

```diff
@@ -1,77 +1,65 @@
 [versions]
-plugin-android = "8.1.1"
-plugin-ktlint = "11.5.1"
-plugin-maven = "0.25.3"
-plugin-multiplatform-compose = "1.6.10"
-plugin-binaryCompatibilityValidator = "0.13.2"
-plugin-atomicfu = "0.23.1"
+agp = "8.7.2"
+kotlin = "2.1.0"
+mavenPublish = "0.30.0"
+composeMultiplatform = "1.7.1"
+binaryCompatibilityValidator = "0.16.3"
+atomicfu = "0.26.1"
```

Comment:
Fyi, this will currently cause an issue because compose expects (and forces) 0.23.2.
Seems to be a known issue https://youtrack.jetbrains.com/issue/CMP-5831